packages/odyssey-react-mui/src/Tag.tsx

```diff
@@ -0,0 +1,41 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Chip } from ".";
+import { memo, useContext } from "react";
+import { TagListContext } from "./TagListContext";
+
+export type TagProps = {
+  isDisabled?: boolean;
+  label: string;
+  onClick?: (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => void;
+  onDelete?: (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => void;
+};
+
+const Tag = ({ isDisabled, label, onClick, onDelete }: TagProps) => {
+  const { chipComponent } = useContext(TagListContext);
+
+  return (
+    <Chip
+      label={label}
+      clickable={onClick ? true : false}
+      component={chipComponent}
+      disabled={isDisabled}
+      onClick={onClick}
+      onDelete={onDelete}
+    />
```

Comment:
Do we want `clickable` based on `onClick`? Doesn't this manage all interactions including deleting? For instance, you don't pass `onClick`, but you pass `onDelete`.

Now what?

What about `onClick || onDelete`?

Reply:
`onClick` and `onDelete` are different in the UI: https://cbea194.ods.dev/?path=/docs/mui-components-tag--default#clickable-tags

`clickable`/`onClick` means the tag itself is clickable and has a function

`onDelete` only applies to the ✖️ that is shown for dismissible tags